### DISCUSSION TITLE
问题已解决——【Question has been solved.】

### DISCUSSION THREAD

**Discussion by rabbit3046** [AlistGo/alist#2287] [General]
Question has been solved. 
Discussions  close.
Discussions  close.
Discussions  close.

**Comment by Becods** [comment]
![CM6PHS7AGGJPN31Y2DI83MS](https://user-images.githubusercontent.com/35525491/201271178-fda8b78d-207c-4e92-9819-49c7cebf5137.png)

**Comment by icexmoon** [comment]
一步步排查，域名解析、apache/nginx、AList

**Reply by icexmoon** [reply]
一般VPS上不建议直接开放其他端口，可以用Apache做反向代理

**Reply by ii1024** [reply]
我现在已经和联通申请了 外网IP ， 怎么实现在外边通过IP：5224 访问家里搭建的alist呢
1 现在已经搭建了alist
2 本地浏览器 127.0.0.1:5244 可以打开
